.travis-command-ea-builds.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 
-set -e
-set +u
+set -euxo pipefail
```

Comment:
I've been following the advice here in all my bash scripts. Let me know if there's a reason not to.

https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/